lib/get_weather.rb

```diff
@@ -40,7 +40,7 @@ def to_s
   end
 
   def self.get_weather(output: $stdout, client: faraday_client, forecast: forecast_passed)
-    response = client.get("/data/2.5/onecall?lat=#{$LAT}&lon=#{$LONG}&appid=#{$APPID}")
+    response = client.get("/data/2.5/onecall?lat=#{$LAT}&lon=#{$LONG}&exclude=hourly,minutely&appid=#{$APPID}")
```

Comment:
This is technically correct but also hard to read. There are two ways to approach cleaning this up:

- put the parameters in a hash and then use `map` and `.join` to collate them
- pass a block to the `get` call and set each parameter using the `params` hash. [Here's an example](https://lostisland.github.io/faraday/usage/#the-connection-object) of setting the `limit` parameter